openwrt-openwisp-monitoring/files/lib/openwisp/interfaces.lua

```diff
@@ -42,7 +42,7 @@ local specialized_interfaces = {
         local signal = io.popen('mmcli --output-json -m '..modem..' --signal-get'):read()
         if signal and pcall(function () signal = cjson.decode(signal) end) then
             -- only send data if not empty to avoid generating too much traffic
-            if not utils.is_table_empty(signal.modem) and not utils.is_table_empty(signal.modem.signal) then
+            if not monitoring.utils.is_table_empty(signal.modem.signal) then
```

Comment:
Changed this as if `signal.modem` is empty then `signal.modem.signal` will point to nil

Reply:
why?

Reply:
I remembered it wrong. It will raise error. Reverting